conda_smithy/cli.py

```diff
@@ -179,14 +179,34 @@ def __init__(self, parser):
         scp = self.subcommand_parser
         scp.add_argument("--feedstock_directory", default=os.getcwd(),
                          help="The directory of the feedstock git repository.")
+        scp.add_argument("-c", "--commit", nargs='?', choices=["edit", "auto"], const="edit",
```

Comment:
(Sorry, I thought I had replied to this earlier)

I say let's go with your gut feeling then, we can change that behavior later if need to. 👍